src/main/java/seedu/address/logic/parser/ArgumentTokenizer.java

```diff
@@ -145,4 +151,41 @@ Prefix getPrefix() {
         }
     }
 
+    /**
+     * Returns List containing user's prefixes.
+     * This function is required since user's can chain prefixes for view.
+     *
+     * @param argsString Input arguments.
+     * @return List containing user's prefixes.
+     */
+    private static List<String> extractPrefixes(String argsString) {
+        Pattern pattern = Pattern.compile("\\b[^\\s]+/");
+        Matcher matcher = pattern.matcher(argsString);
+        List<String> prefixes = new ArrayList<>();
+        while (matcher.find()) {
+            prefixes.add(matcher.group().trim());
+        }
+        System.out.println(prefixes);
+        return prefixes;
+    }
+
+    /**
+     * Returns ArgumentMultimap object
+     * Converts user's chained prefixes into a list.
+     *
+     * @param argsString User's input.
+     * @return ArgumentMultimap object.
+     * @throws ParseException if duplicate prefixes are found.
+     */
+    public static ArgumentMultimap viewTokenize(String argsString) throws ParseException {
+        List<String> extractedPrefixes = extractPrefixes(argsString);
+
+        Set<String> uniquePrefixes = new HashSet<>(extractedPrefixes);
+        if (uniquePrefixes.size() != extractedPrefixes.size()) {
+            throw new ParseException("Duplicate prefix detected in input.");
```

Comment:
Same issue here, can consider adding more test cases for failure in ArgumentTokenizerTest.java